src/chrome/content/rules/Inc.com.xml

```diff
@@ -1,68 +1,19 @@
 <!--
-	For rules causing MCB, see Inc.com-falsemixed.xml.
-
-
-	Nonfunctional subdomains:
-
-		- admin ¹
-		- dev-admin ²
-		- store ¹
-
-	¹ Shows CentOS test page
-	² Refused
-
-
-	These altnames don't exist:
-
-		- www.subscriptions.inc.com
-
-
-	Mixed content:
-
-		- frame on www from dev-admin ¹
-
-		- css, on:
-
-			- conference from $self ²
-			- conference and www from netdna.bootstrapcdn.com ²
-			- conference from fonts.googleapis.com ²
-			- mediakit from $self ²
-			- www from $self ²
-			- www from admin ¹
-
-		- Images, on:
-
-			- www from admin ¹
-			- www from dev-admin ¹
-
-	¹ Unsecurable
-	² Secured by us
+	Non-functional hosts
+		SSL peer certificate was not OK:
+		- subscriptions.inc.com
 
+		Mixed content blocking (MCB) triggered:
+		- mediakit.inc.com
 -->
-<ruleset name="Inc.com (partial)">
-
+<ruleset name="Inc.com">
 	<target host="inc.com" />
-	<target host="*.inc.com" />
-		<!--
-			Avoid broken MCB:
-						-->
-		<exclusion pattern="^http://(?:conference|mediakit)\.inc\.com/+(?!wp-content/|wp-includes/)" />
-		<exclusion pattern="^http://(?:www\.)?inc\.com/+(?!favicon\.ico|lib/|views/css/)" />
-
+	<target host="www.inc.com" />
+	<target host="conference.inc.com" />
```

Comment:
For the record, this target has MCB but the server redirects to HTTPS so I'm fine with keeping it.